src/main/java/org/apache/ibatis/io/ClassLoaderWrapper.java

```diff
@@ -185,9 +185,7 @@ URL getResourceAsURL(String resource, ClassLoader[] classLoader) {
 
           Class<?> c = Class.forName(name, true, cl);
 
-          if (null != c) {
-            return c;
-          }
+          return c;
```

Comment:
We can also remove the temporary variable `c`.

```java
return Class.forName(name, true, cl);
```

Reply:
you are right. changed.

Reply:
> We can also remove the temporary variable `c`.
> 
> ```java
> return Class.forName(name, true, cl);
> ```

you are right. changed.